src/sparse/impl/KokkosSparse_gauss_seidel_spec.hpp

```diff
@@ -110,19 +110,19 @@ namespace KokkosSparse {
                                             KokkosKernels::Experimental::KokkosKernelsHandle< \
                                                                                              const OFFSET_TYPE, const ORDINAL_TYPE, const SCALAR_TYPE, \
                                                                                              EXEC_SPACE_TYPE, MEM_SPACE_TYPE, SLOW_MEM_SPACE> , \
-                                            Kokkos::View<const OFFSET_TYPE *, LAYOUT_TYPE, \
+                                            Kokkos::View<const OFFSET_TYPE *, Kokkos::LayoutLeft, \
```

Comment:
There should be no need to specialize on LayoutLeft vs. LayoutRight for rank-1 Views.  Kokkos::View permits free assignment between LayoutLeft and LayoutRight in that case.  We should thus just be able to leave off the layout and let the default take over.

Reply:
@mhoemmen Right, before this PR the ETI unnecessarily instantiated apply for both Left and Right layout rowptrs/colinds/values. I always just take a LayoutLeft unmanged view of those now. I changed the ETI so that LAYOUT_TYPE is the layout of X and Y, since those are now rank-2.